Makefile

```diff
@@ -95,16 +95,26 @@ hugo/run:
 run: docker/build
 	$(DOCKER_RUN) hugo/run
 
+## Build customized utterances widget
+utterances/build:
+	rm -rf utterances
+	git clone --branch master https://github.com/cloudposse/utterances.git
+	cd utterances && yarn && yarn build
+	mkdir -p public/utterances
```

Comment:
I don't think you want this `mkdir` here since below you're moving (renaming) one directory to this name.